openedx/adg/lms/webinars/models.py

```diff
@@ -166,6 +170,21 @@ def save(self, force_insert=False, force_update=False, using=None, update_fields
     def webinar_team(self):
         return set(chain(self.co_hosts.all(), self.panelists.all(), {self.presenter}))
 
+    def remove_team_registrations_and_cancel_reminders(self, removed_members):
+        """
+        Given a list of team members of the webinar, remove their team registrations and cancel all reminder emails
+
+        Args:
+            removed_members (list): List of team members whose registrations are to be removed and reminders cancelled
+
+        Returns:
+            None
+        """
+        WebinarRegistration.remove_team_registrations(removed_members, self)
```

Comment:
We do not really need a class function anymore.
We can add a simple model method that removes the registrations of the given members.

We can do something like `self.registrations.remove_team_registrations(removed_members)`

Reply:
Also if we cancel all reminders every time the registrations are removed, we can move the `cancel_all_remainders` to that model method as well

Reply:
@MunamMubashir-arbisoft any specific reason behind replacing `WebinarRegistration. remove_team_registrations ` with `self.registrations. remove_team_registrations`. At the end both would be calling the class method, no? Btw, that would require updating the `remove_team_registrations` function as well, as it is currently receiving webinar as a parameter.

Reply:
`cancel_all_reminders` is being called at other places as well, like `api_views.py` and `tasks.py`.

Reply:
I actually meant that we can remove the class method altogether. And add a model method (not a class method) instead.